src/sentry/rules/conditions/event_frequency.py

```diff
@@ -29,6 +29,10 @@
 }
 COMPARISON_TYPE_COUNT = "count"
 COMPARISON_TYPE_PERCENT = "percent"
+comparison_types = {
+    COMPARISON_TYPE_COUNT: "Absolute increase",
+    COMPARISON_TYPE_PERCENT: "Percent increase",
+}
```

Comment:
@taylangocmen Not sure if you'll even be using these values for display. If you want to, let me know and I'll just update them to whatever you need.

Reply:
[The fe designs for this changed a bit](https://www.figma.com/file/AQwJvc18VjSWWc3ujAWt4O/Alerts-V3?node-id=360%3A20081) handling these in the fe will be easier in my opinion, I have a pr draft up for it